code/src/sixsq/nuvla/server/resources/module.clj

```diff
@@ -168,6 +168,17 @@ component, or application.
       second
       first))
 
+
+(defn count-children-as-admin [parent-path]
```

Comment:
I could not find an existing fn to check for children, so I wrote this one.
I was thinking to put it under `sixsq.nuvla.server.resources.module.utils` but I cannot reference `resource-type` from there. Is there a better way?

Reply:
Perfect. The resource-type is a bit annoying because it correspond to the *ns*. I don't have the right answer to it. Maybe defining resource-type as a constant string in utils and refering to it in module namespace instead of *ns* is better. Will let you decide what's better

Reply:
Added a constant to the utils ns. A bit of repetition, but shouldn't hurt too much.
I haven't changed the constant in the module ns for now, maybe its good to keep a strong link with the ns name? I can think of it later on, as I learn more.